### PR TITLE
get the tests passing on CI

### PR DESCRIPTION
**Pull request by davisjam**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "repository": "https://github.com/segmentio/is-url",
   "version": "1.2.3",
   "scripts": {
-    "test": "make test"
+    "test": "./node_modules/.bin/mocha --reporter spec"
```
Comment (zeke) [inline package.json]: npm adds everything in `./node_modules.bin` to the PATH at runtime, so there's no need for the fully qualified path. Just `mocha` or `mocha --reporter spec` will suffice.

**Comment by zeke** [comment]
Oh can you also add a `.travis.yml`? The following content should suffice:

```
language: node_js
node_js:
  - 8
```

**Comment by davisjam** [comment]
@zeke Travis added in a4e77ae. CI is still unhappy, please advise.

**Comment by zeke** [comment]
Fixed in https://github.com/segmentio/is-url/pull/23

I think this PR was not passing because forked repos are not allowed to add/modify the travis config.

**Pull request closed without merging**